cmd/argocd/commands/app.go

```diff
@@ -747,6 +750,9 @@ func NewApplicationSetCommand(clientOpts *argocdclient.ClientOptions) *cobra.Com
   # Set and override application parameters with a parameter file
   argocd app set my-app --parameter-file path/to/parameter-file.yaml
 
+  # Set and override application parameters for a source at index 1 under spec.sources of app my-app (Indexes start at 0)
```

Comment:
I was wondering if it would confusing be for the majority of users that the index starts at 0. For programmers, it might feel natural to start counting at 0, but the majority of people probably start counting at 1 :slightly_smiling_face:

Reply:
Updated it to default at 1 😅